nibabel/streamlines/trk.py

```diff
@@ -283,10 +286,10 @@ def _default_structarr(cls):
         return st_arr
 
     @classmethod
-    def create_empty_header(cls):
+    def create_empty_header(cls, endianness=None):
```

Comment:
I added this in passing, since it's the only other place `_default_structarr` is being called.